Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol ConsentTransparencyViewDelegate: AnyObject {
+    func consentTransparencyView(_ consentTransparencyView: ConsentTransparencyView, didSelectMore button: Button)
+    func consentTransparencyView(_ consentTransparencyView: ConsentTransparencyView, didSelectOkay button: Button)
+}
+
+public final class ConsentTransparencyView: UIView {
+
+    // MARK: - Internal properties
+
+    private let topImage = UIImage(named: .consentTransparencyImage)
+
+    private lazy var imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFit
+        imageView.image = topImage
+        return imageView
+    }()
+
+    private lazy var headerLabel: Label = {
+        let label = Label(style: .title3)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Din data, dine valg"
+        return label
+    }()
+
+    private lazy var detailLabel: Label = {
+        let label = Label(style: .detail(.licorice))
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.text = "FINN er en del av Schibsted Norge. Når du bruker FINN er Schibsted Norge behandlingsansvarlig for påloggingsløsning og reklame, mens FINN er behandlingsansvarlig for det øvrige innholdet i tjenesten vår. Både FINN og Schibsted Norge behandler data om deg.\n\nFINN bruker dine data til å tilpasse tjenestene til deg, mens Schibsted Norge i tillegg bruker dem til å gi deg mer relevante annonser. Persondata brukes også for å sikre at tjenestene er trygge og sikre for deg."
```

Comment:
We add these texts from a model in other components. Now they are hardcoded. Any reason not to follow that pattern?